bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/DPIUtil.java

```diff
@@ -347,6 +347,18 @@ public static int getZoomForAutoscaleProperty (int nativeDeviceZoom) {
 	return getZoomForAutoscaleProperty(nativeDeviceZoom, autoScaleValue);
 }
 
+public static int getNativeZoomForAutoscaleProperty (int nativeDeviceZoom) {
```

Comment:
I find this method name a bit confusing. From it's name, I would expect the method to rather return the argument, as passing a native zoom to get a native zoom should be kind of a no-op. Isn't this rather a `getFontZoomForAutoscaleProperty()`, as it is supposed to return the zoom to be used for fonts based on the given native zoom?

Reply:
I have renamed this